src/sentry/eventstore/models.py

```diff
@@ -769,7 +769,11 @@ def __getitem__(self, name: str) -> str:
         elif name == "orgID":
             return cast(str, self.event.organization.slug)
         elif name == "title":
-            return self.event.title
+            return (
+                self.event.occurrence.issue_title
+                if hasattr(self.event, "occurrence")
```

Comment:
I think this needs to be `getattr(self.event, "occurrence", None)`, otherwise this will always evaluate to true for `GroupEvent` instances.